logstash-core/property(org.gradle.api.file.Directory, fixed(class org.gradle.api.internal.file.DefaultFilePropertyFactory$FixedDirectory, /Users/rye/src/elastic/logstash@main/logstash-core/build))/generated/version_info.properties

```diff
@@ -0,0 +1 @@
+version=8.16.0
```

Comment:
Ah, it seems accidentally pushed from your local.

Reply:
Whoops. This was a leftover from gradle attempts that accidentally got included.